crates/optimism/node/src/node.rs

```diff
@@ -747,32 +744,46 @@ where
 
 /// A basic optimism network builder.
 #[derive(Debug, Default, Clone)]
-pub struct OpNetworkBuilder {
+pub struct OpNetworkBuilder<NetworkPrimitives = OpNetworkPrimitives, PooledTx = OpPooledTransaction>
+{
```

Comment:
let's rename the generic `NetworkPrimitives` to just `NetP`, so that this doesnt clash with the NetworkPrimitives trait def